web/src/api/InventoryAPI.js

```
@@ -82,5 +82,25 @@ const useGetInventory = (playerId, experienceId) => {
     return { inventory, loading, error };
 };
 
+const useUpdateInventoryContext = (context) => {
+    const { getAccessTokenSilently } = useAuth0();
+    if (!context || !context.inventory) {
+        return;
+    }
+
+    const getUrl = `${BACKEND_URL}/inventory/${context.inventory.inventory_id}`;
+    
+    const updateInventoryContext = async () => {
+        const token = await getAccessTokenSilently();
+        try {
+            context.setInventory(await fetchData(getUrl, token));
+        } catch (error) {
+            console.log(error)            
```

Can we fail silently? Exposing the error in the log will allow to obtain the token